certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -4784,83 +5013,183 @@
-              "type": "string",
-              "description": "The source SharePoint Library/List ID",
-              "title": "Library/List ID",
-              "x-ms-visibility": "important"
-            },
-            "libraryName": {
-              "type": "string",
-              "description": "The source SharePoint Library/List Name",
-              "title": "Library/List Name",
-              "x-ms-visibility": "important"
-            },
-            "item": {
-              "type": "integer",
-              "format": "int32",
-              "description": "The selected document/item ID, or the first selected document/item ID from the items array.",
-              "title": "Item",
-              "x-ms-visibility": "important"
-            },
+        "x-ms-visibility": "internal",
+        "summary": "Get an Operation Status for GetTrackedChanges",
+        "description": "Retrieve the operation status of a GetTrackedChanges operation for the Operation ID provided."
+      }
+    },
+    "/api/v1/Mgmt/GetOperationStatusAddWatermark": {
+      "get": {
+        "tags": [
+          "Mgmt"
+        ],
+        "operationId": "GetOperationStatusAddWatermark",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "operationId",
+            "description": "Retrieve the operation status of an AddImageWatermark, AddImageWatermarkAdv, AddTextWatermark or AddTextWatermarkAdv operation for the Operation ID provided.",
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": { }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DtoResponseAddImageWatermark"
+            }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/DtoResponseAddImageWatermark"
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {
+              "$ref": "#/definitions/DtoResponseAddImageWatermark"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
+        "summary": "Get an Operation Status for Add Watermark",
+        "description": "Get the operation status of for the operation ID provided."
+      }
+    },
+    "/api/v1/Trigger/ManageWebHook": {
+      "post": {
+        "tags": [
+          "Trigger"
+        ],
+        "operationId": "ManageWebHookAsync",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "request",
+            "description": "",
```

Comment:
@Encodian can you please explain why you have empty descriptions? Please define a proper description.

Reply:
@Amjed-Ayoub - This is not a UI specific action, its part of a Trigger action. This hasn't changed since the previously approved version, we auto-generate the swagger definition in code so it is not a quick fix. We can raise a ticket to address but would request this is approved as it matches the current approved version, is this possible?

Reply:
Resolved with update